Support/Testing/Travis/before-script.sh

```diff
@@ -20,7 +20,7 @@ if [[ "$TARGET" == Android-* && -n "${LLDB_TESTS-}" ]]; then
 
   case "${TARGET}" in
     "Android-ARM") emulator_arch="arm";;
-    "Android-ARM64") emulator_arch="aarch64";;
+    "Android-ARM64") emulator_arch="ranchu-arm64";;
```

Comment:
Ranchu is the machine type used when invoking qemu-android-aarch64 or whatever it's called. I wish it was just `emulator64-arm64` with the default qemu argument being `-machine type=ranchu`. Including it in the name of the emulator binary feels strange. :/